apps/condo/domains/billing/schema/BillingReceipt.js

```diff
@@ -106,6 +106,27 @@ const BillingReceipt = new GQLListSchema('BillingReceipt', {
             kmigratorOptions: { null: true, on_delete: 'models.CASCADE' },
         },
 
+        invalidServicesError:  {
+            schemaDoc: 'Indicates if services are valid and add up to total sum toPay.',
+            type: Virtual,
+            isRequired: false,
+            resolver: async (item) => {
+                let servicesTotal = 0
+
+                if (get(item, 'services')){
+                    for (let service of get(item, 'services')){
+                        servicesTotal = Big(get(service, 'toPay')).plus(Big(servicesTotal))
+                    }
+                }
+
+                const isServicesValid = Big(servicesTotal).cmp(Big(get(item, 'toPay'))) === 0
```

Comment:
servicesTotal is already a `Big` type value, so you might not need to cast it to `Big` here

Reply:
Also toPay might be initialized just after the `servicesTotal` initialization line. So it would look like this:

```
let servicesTotal = Big(0)
const toPay = Big(get(item, 'toPay')

...

// not sure if this will work, but i think it looks better this way
const servicesAreValid = servicesTotal.cmp(toPay) === 0

```